examples/zkapps/anonymous-message-board/src/index.ts

```diff
@@ -1,74 +1,3 @@
-import {
-  Field,
-  PrivateKey,
-  SmartContract,
-  state,
-  State,
-  method,
-  PublicKey,
-  Poseidon,
-  isReady,
-} from 'snarkyjs';
+import { Add } from './Add.js';
```

Comment:
we should remove this as well if we decide to remove 'Add.ts` and `Add.test.ts` - they are just dead weight

Reply:
Okay just to be clear you mean we should remove the 'Add.ts' and Add.test.ts files right?

Reply:
yep, I think we should remove all boilerplate code thats not used/needed